.eslintignore

```diff
@@ -0,0 +1,5 @@
+**/node_modules/*
```

Comment:
I can't seem to find your README.md on this PR, but I did manage to pull it down, so just a reminder here to write your README per the lab instructions too. :)